### PR TITLE
[Calling] Hide active speakers/all toggle

### PR DESCRIPTION
**Pull request by mejdoo**
## What's new in this PR?

The calling squad decided to disable all/speaker toggle to perform some optimizations in terms of user experience before releasing it to public. The code is now commented till further notice.
#### APK
[Download build #3093](http://10.10.124.11:8080/job/Pull%20Request%20Builder/3093/artifact/build/artifact/wire-dev-PR3154-3093.apk)

### CODE REVIEW

File: app/src/main/scala/com/waz/zclient/calling/ControlsFragment.scala [hunk 1]
```diff
@@ -94,21 +94,22 @@ class ControlsFragment extends FragmentHelper {
...
-      }
-    }
-    else {
-      speakersLayoutContainer.foreach(_.setVisibility(View.INVISIBLE))
-    }
-
+    //TODO : The calling squad decided to disable all/speaker toggle to perform some optimizations
+    // in terms of user experience before releasing it to public
+
+    /* if (BuildConfig.ACTIVE_SPEAKERS) {
```
Comment (gizemb) [inline app/src/main/scala/com/waz/zclient/calling/ControlsFragment.scala:100]: why didn't you just change the flag?

Comment (mejdoo) [reply app/src/main/scala/com/waz/zclient/calling/ControlsFragment.scala:100]: Because we want to release the active speaker feature except this toggle.

**Review by makingthematrix — APPROVED** [review]

File: app/src/main/scala/com/waz/zclient/calling/ControlsFragment.scala (hunk 1 above)
Comment (makingthematrix) [reply app/src/main/scala/com/waz/zclient/calling/ControlsFragment.scala:100]: @mejdoo : Please comment out also the unused imports.

Comment (mejdoo) [reply app/src/main/scala/com/waz/zclient/calling/ControlsFragment.scala:100]: ok good catch @makingthematrix

**Pull request merged**